cli.js

```diff
@@ -219,6 +222,12 @@ async function restoreUsers() {
     const response = await cognitoIsp.send(new AdminCreateUserCommand(params));
     debug('Restored user', response?.User?.Username);
 
+    if (verbose) {
+      const oldSub = user.Attributes.find((attribute) => attribute.Name === 'sub');
+      const newSub = response?.User.Attributes.find((attribute) => attribute.Name === 'sub');
+      console.log(`Restored user - oldSub: "${oldSub.Value}" newSub: "${newSub.Value}"`);
```

Comment:
I don't trust them 😅
```suggestion
      const newSub = response.User.Attributes.find((attribute) => attribute.Name === 'sub');
      console.log(`Restored user - oldSub: "${oldSub?.Value}" newSub: "${newSub?.Value}"`);
```